docs/changehistory/NextVersion.md

```diff
@@ -29,6 +29,12 @@ This feature is currently disabled by default. Enabling it requires the use of A
    IModelApp.startup({ tileAdmin });
 ```
 
+## Change of default behavior of `IModelDb.close()` and `IModelDb.saveChanges()`
+
+By default `IModelDb.close()` use to save changes. But now it does not. Application must call `IModelDb.saveChanges()` explicitly before closing to ensure any changes made so far are committed to disk. `IModelDb.saveChanges()` also now expect to throw an exception if it fails and rollback any changes. Application code must treat exceptions as fatal an should exit afterwords. Application should not expect this to be common case and `IModelDb.saveChanges()` only fail in rare cases where there were not enough system resources to complete the operation.
```

Comment:
Can you clarify what happens if user closes the iModel with uncommitted changes?

Reply:
Thanks. Do you mind if I edit this for spelling and clarity?

Reply:
sure.